switchmap/poller/snmp/mib/cisco/mib_ciscovtp.py

```
@@ -84,25 +85,28 @@ def layer2(self):
+            self.vtpvlanstate(),
+            self.vtpvlantype(),
+            return_exceptions=True,
+        )
 
-        # Get interface vtpVlanState data
-        values = self.vtpvlanstate()
-        for key, value in values.items():
-            final[key]["vtpVlanState"] = value
+        method_names = ["vtpVlanName", "vtpVlanType", "vtpVlanState"]
+
+        for i, (method_name, values) in enumerate(zip(method_names, results)):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)